app/views/dashboard/show.html.erb

```diff
@@ -56,4 +56,20 @@
       <% end %>
     </table>
   </div>
+  <%= link_to "New Case Contact",  new_case_contact_path %>
+
+  <table class="case-contacts-table">
+    <tr>
+      <th>Date</th>
+      <th>Duration</th>
+      <th>Type</th>
+    </tr>
```

Comment:
I'm wondering if in this table we want to list the case number of the `CasaCase`. Judging by the `actively_assigned_to` scope, I feel like it's combining all `CaseContacts`, potentially mixing them between multiple `CasaCase`s. Or am I not reading that right? (quite probable!) I _think_ while it is in reality rare, there is the possibility that a `Volunteer` is assigned to multiple active `CasaCase`s at once.

Reply:
Oh, this is a good point

Reply:
Cool! Let me know when ready for final review!

Reply:
How do you feel about moving this out into its own issue?

Reply:
I feel great about that! Shall I go ahead and rebase and merge this puppy? I can also write up the other issue.